packages/main/src/plugin/kubernetes/kubernetes-port-forward-service.ts

```diff
@@ -79,6 +79,10 @@ export class KubernetesPortForwardService implements IDisposable {
     this.#disposables.forEach(disposable => disposable.dispose());
```

Comment:
not sure this works anymore, as the `disposables` is now a Map instead of an array, maybe we need something like `Array.from(this.#disposables.values()).forEach(disposable => disposable.dispose());`